crates/anvil/src/eth/backend/cheats.rs

```diff
@@ -1,7 +1,8 @@
 //! Support for "cheat codes" / bypass functions
 
+use alloy_primitives::Address;
+use alloy_rpc_types::Signature;
```

Comment:
@DaniPopes are we mixing types here? or should we pull signature into primitives?

Reply:
We do have `alloy_primitives::Signature`! we'll pull it in once needed when we get into actually moving types that can recover their signer (`TypedTransaction`, e.g) here #6778. For now we can use this type, as right now we just need an equivalence check